### PR TITLE
Add script to quickly build docker images locally

### PR DESCRIPTION
**Pull request by sveitser**
@nomaxg This isn't urgent but when things calm down (or if you need it), could you test the script on Intel OSX? 

For me this works to generate a runnable docker image on NixOS and M1 Mac.

### CODE REVIEW

**Comment by jbearer** [comment]
This successfully build the Rust binaries but then I got `scripts/build-docker-images-native: line 77: patchelf: command not found`. I suppose I could just install `patchelf`. But could we add that to the regular Nix shell (the one that doesn't take hours to build)?

**Comment by sveitser** [comment]
Thanks for trying. Yes we should add patchelf to our default dev shell in the flake. Not sure why I have it on my system. Maybe a nixos thing.

**Comment by sveitser** [comment]
Ah. I think the script is wrong. I wanted to build in docker on OSX which wouldn't require patchelf. Afaik on OSX without cross compiling we get mach-o executables otherwise which won't work in docker.

File: scripts/build-docker-images-native [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+trap cleanup EXIT
+cleanup(){
+    rm -rfv ${WORKDIR}
+}
+
+mkdir -p ${WORKDIR}/target/$ARCH/release
+for binary in "orchestrator" "web-server" "sequencer" "example-l2" "cli"; do
+  cp -v target/release/$binary ${WORKDIR}/target/$ARCH/release
+  # Patch the interpreter for running without nix inside the ubuntu based docker image.
+  if [ $ARCH == "amd64" ]; then
```
[the remark this answers is not in the document]
Comment (jbearer) [reply scripts/build-docker-images-native:76]: It works with this change!

**Review by jbearer — APPROVED** [review]

**Pull request merged**